applications/GeoMechanicsApplication/tests/test_line_interface_elements.py

```diff
@@ -81,6 +82,68 @@ def assert_outputs_for_3_plus_3_line_interface_element(self, simulation):
             self.assertAlmostEqual(relative_displacement_vertical_element[index][1], expected_vertical_displacement)
 
 
+    def assertVectorsAlmostEqual(self, actual_vectors, expected_vectors):
```

Comment:
Good catch. It is more general indeed. However, it can't be moved to, e.g. `test_helper.py`, since its implementation calls another assertion method, which is only known to test case instances. In other words, this checker must be part of the test case class. Perhaps at some point in time it can be moved to the Kratos Core...?

I did see room for one more improvement: I have found that class `TestCase` has a method `assertVectorAlmostEqual`. By reusing this method, I was able to get rid of the inner loop.